metaswitch/common/test/test_extended_alarm.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
I can't remember what this means.
The test that uses the above JSON file is testExtendedValidFile, this checks the the JSON parser has correctly parsed the name, index and two severities of the alarm. I don't think there are any wrong JSON names.